app/Models/Schedule.php

```diff
@@ -103,6 +103,7 @@ class Schedule extends Model implements HasPresenter
         'name'         => 'required|string',
         'message'      => 'nullable|string',
         'status'       => 'required|int|between:0,2',
+        'completed_at' => 'after_or_equal:scheduled_at',
```

Comment:
I didn't know about `after_or_equal`, that's cool! Should probably be `nullable` though (see above comment).

Reply:
Cool :) Will keep this, and based on our discussion above I'll add that